src/Config/Converters/DataSourceConverterFactory.cs

```diff
@@ -47,7 +47,7 @@ public DataSourceConverter(bool replaceEnvVar)
 
         public override DataSource? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
-            DataSource dataSource = new(DatabaseType.MSSQL, string.Empty, null);
+            DataSource dataSource = new(DatabaseType.MSSQL, string.Empty, null, new());
```

Comment:
lets make sure even though when null means enabled and we instantiate the `health` property, it doesn't end up writing to the file system if the original configuration did not have it when performing `dab add` or `dab update` commands.

1) create config with no health config section
2) dab add entity -> the modified config that gets written to the file system should not add the health section when writing to the file system.